uint8 NUM_ACTUATOR_CONTROLS = 8
uint8 NUM_ACTUATOR_CONTROL_GROUPS = 4
uint64 timestamp_sample	    # the timestamp the data this control response is based on was sampled
float32[8] control
